Makefile

```diff
@@ -127,6 +127,52 @@ restart-celery:
 		sudo systemctl status --no-pager celeryworker.service) & \
 	fi
 
+check-local-hub:
+	@echo "Checking LOCAL_HUB setting..."
+	@if [ "$$(uname)" = "Darwin" ]; then \
+		grep -E '^LOCAL_HUB=' .env | cut -d'=' -f2 | tr -d '"' | tr -d ' ' > .hub_setting; \
```

Comment:
should `.hub_setting` be git-ignored?

Reply:
thanks. have merged this into the main `restart-hub` function